Repositories/PostMachineRepository.cs

```diff
@@ -1,32 +1,34 @@
-﻿using shipping_service.Persistence.Entities;
+﻿using shipping_service.Persistence.Database;
+using shipping_service.Persistence.Entities;
 
 namespace shipping_service.Repositories
 {
     public class PostMachineRepository : IPostMachineRepository
     {
-        public async Task<List<PostMachine>> GetAsync()
-        {
-            return new List<PostMachine>();
-        }
+        private readonly DatabaseContext context;
 
-        public async Task<PostMachine> GetAsync(ulong id)
+        public PostMachineRepository(DatabaseContext ctx)
         {
-            return new PostMachine();
+            context = ctx;
         }
 
-        public async Task<PostMachine> CreateAsync(PostMachine postMachine)
+        public IQueryable<PostMachine> PostMachines => context.PostMachines;
+
+        public async Task CreateAsync(PostMachine postMachine)
         {
-            return new PostMachine();
+            await context.AddAsync(postMachine);
+            await context.SaveChangesAsync();
         }
 
-        public async Task<PostMachine> UpdateAsync(ulong id, PostMachine postMachineIn)
+        public async Task UpdateAsync(PostMachine postMachine)
         {
-            return new PostMachine();
+            await context.SaveChangesAsync();
         }
 
-        public async Task<ulong> DeleteAsync(ulong id)
+        public void DeleteAsync(PostMachine postMachine)
```

Comment:
This should be either be made async, or remove Async in the name for less confusion.